packages/contractkit/src/identity/offchain-data-wrapper.ts

```diff
@@ -6,14 +6,21 @@ import { ContractKit } from '../kit'
 import { ClaimTypes } from './claims/types'
 import { IdentityMetadataWrapper } from './metadata'
 import { StorageWriter } from './offchain/storage-writers'
+
 const debug = debugFactory('offchaindata')
 
+class FetchError extends Error {}
+class InvalidSignature extends Error {}
+class NoStorageRootProvidedData extends Error {}
+export type OffchainErrors = FetchError | InvalidSignature | NoStorageRootProvidedData
+export type ReadResponse = { status: 'ok'; data: any } | { status: 'error'; error: OffchainErrors }
```

Comment:
How about an enum for these status values instead of using raw strings here?

Reply:
Great point, added that to the Task type